site/src/pages/components/selectlist.mdx

```diff
@@ -59,18 +59,24 @@ Selectlist provides a variety of tools to help with styling, including pseudo-se
 
 ### Styling parts of the control
 
+Selectlist supports a variety of pseudo-elements to target its different parts:
+- `::button` - Targets the button slot.
+- `::listbox` - Targets the `<listbox>`.
+- `::marker` - Targets the dropdown arrow of the default supplied button.
```

Comment:
Are these psuedo elements only available if the author doesn't provide their own element? At which point I should be able to simply style them as elements, correct?

Reply:
They should target whatever is rendered, whether thats the default/fallback content or the author supplied one. I added a sentence to clarify